src/core/overlay/position/global-position-strategy.ts

```diff
@@ -0,0 +1,87 @@
+import {PositionStrategy} from './position-strategy';
+import {DOM} from '../../platform/dom/dom_adapter';
+
+
+/**
+ * A strategy for positioning overlays. Using this strategy, an overlay is given an
+ * explicit position relative to the browser's viewport.
+ */
+export class GlobalPositionStrategy implements PositionStrategy {
+  private _cssPosition: string = 'absolute';
+  private _top: string = '';
+  private _bottom: string = '';
+  private _left: string = '';
+  private _right: string = '';
+
+  /** Array of individual applications of translateX(). Currently only for centering. */
+  private _translateX: string[] = [];
+
+  /** Array of individual applications of translateY(). Currently only for centering. */
+  private _translateY: string[] = [];
+
+  fixed() {
+    this._cssPosition = 'fixed';
+    return this;
+  }
+
+  absolute() {
+    this._cssPosition = 'absolute';
+    return this;
+  }
+
+  top(value: string) {
+    this._top = value;
+    return this;
+  }
+
+  left(value: string) {
+    this._left = value;
```

Comment:
You should set `this._right = ''` and `this._translateX = []` to counter `.center(...).left(...)` and `.right().left()`. Same with others.